docs/OnnxReleases.md

```diff
@@ -5,7 +5,7 @@ The ONNX project, going forward, will plan to release roughly on a four month ca
 ## Preparation
 
 * Install Twine, a utility tool to interact with PyPI. Do  - ``pip install twine``
-* Get hold of the username and password for the ‘onnx’ PyPI account. Release manager should get onnx pypi account credentials from steering committee or from previous release manager.
+* Get hold of the username and password for the ‘onnx’ PyPI account. Release manager should get onnx PyPI account credentials from steering committee or from previous release manager.
```

Comment:
"get onnx PyPI account credentials from steering committee or from previous release manager" -> "get ONNX PyPI account credentials from the credential repository (ask Steering Committee for access)"